packages/dnb-eufemia/src/extensions/forms/Iterate/Toolbar/Toolbar.tsx

```diff
@@ -1,13 +1,85 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import classnames from 'classnames'
 import { Hr } from '../../../../elements'
 import { Flex, Space } from '../../../../components'
 import { SpaceAllProps } from '../../../../components/Space'
+import IterateItemContext from '../IterateItemContext'
+import ElementBlockContext from '../AnimatedContainer/ElementBlockContext'
+import EditButton from '../ViewContainer/EditButton'
+import RemoveButton from '../ViewContainer/RemoveButton'
+import CancelButton from '../EditContainer/CancelButton'
+import DoneButton from '../EditContainer/DoneButton'
 
-export type Props = SpaceAllProps
+export type ToolbarParams = {
+  index: number
+  items: Array<unknown>
+  EditButton: typeof EditButton
+  RemoveButton: typeof RemoveButton
+  CancelButton: typeof CancelButton
+  DoneButton: typeof DoneButton
+}
+export type Props = Omit<SpaceAllProps, 'children'> & {
+  children?: React.ReactNode | ((params: ToolbarParams) => React.ReactNode)
+  variant?: /**
+   * Use this variant to hide the toolbar when there is only one item in the array.
+   */
+  'minimumRequiredItems'
```

Comment:
Based on the descriptive line above, could this be a better more understandable prop name?

```suggestion
  'hidesWhenOneItem'
```

Personally, I still don't understand the name/word `minimumRequiredItems`.
I'll try to better understand this feature, and come up with a different name.

Maybe you can try to explain to me what happens when I as a user use the variant `minimumRequiredItems`?


```suggestion
  'theusecase'
```

By theusecase, I mean maybe the variant name should be "the use case", in which use case would I want to use this variant?

Reply:
Yes, lets find a meaningful variant or prop name.

The variant changes have been moved over to this PR #3919
This PR is only for the new container behavior and the fully customization of a iterate Toolbar.